src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,16 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { PostInfo } from '../PostInfo';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  posts: Post[]
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <div className="PostList">
+
+    {posts.map(post => (
+      <PostInfo post={post} />
+    ))}
```

Comment:
whenever mapping anything in JSX - `key` prop is a must
read about React reconciliation (https://reactjs.org/docs/reconciliation.html#keys) to better feel why keys are important